modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/DatabaseRegistry.java

```diff
@@ -207,6 +207,7 @@ void checkDatabases(ClusterState state) {
             String remoteMd5 = metadata.getMd5();
             String localMd5 = reference != null ? reference.getMd5() : null;
             if (Objects.equals(localMd5, remoteMd5)) {
+                reference.setLastUpdate(metadata.getLastUpdate());
```

Comment:
I don't think that we have yet a test for this in `DatabaseRegistryTests`, maybe add this and check that last update time is updated?

Reply:
I've updated `DatabaseRegistryTests#testCheckDatabases` because setup would be almost identical for separate test